docs/protocol/extension_statistics.md

```diff
@@ -325,7 +350,16 @@ message ModelStatistics
   // executed in the model. The batch statistics indicate how many actual
   // model executions were performed and show differences due to different
   // batch size (for example, larger batches typically take longer to compute).
-  InferBatchStatistics batch_stats = 7;
+  repeated InferBatchStatistics batch_stats = 7;
```

Comment:
Why does this need to be repeated? Will this affect the tests that check this (e.g. L0_batcher).

Reply:
The field itself is repeated, just the doc here is not matching the service definition. It is repeated because each entry captures the stats for a batch size, so there will be multiple entries for multiple different batch sizes.

Reply:
Got it, thanks for explaining.